src/components/CardModal.js

```diff
@@ -70,12 +73,18 @@ export default function CardModal({
               <CardBadge>Area Control</CardBadge>
               <CardBadge>Route/Network Building</CardBadge>
             </CardCategories>
-            <AddButtonCollection handleAddButton={onCollectionButton}>
-              <Star />
-            </AddButtonCollection>
-            <AddButtonWishlist handleAddButton={onWishlistButton}>
-              <Dice />
-            </AddButtonWishlist>
+            {enabled && (
+              <AddButtonCollection
+                handleClick={() => addGameToCollection(singleGame)}
```

Comment:
You could be a bit more specific with what is actually handled here ( e.g."handleAdd"). I think you were the one mentioning this tipp to me the other day so I hereby give it back to you ;-)